packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -61,10 +65,11 @@ const DEFAULT_IOS_LONG_PRESS_MIN_DURATION =
  *
  * @param {Object}      props          Component props.
  * @param {JSX.Element} props.children Children to be rendered.
+ * @param {boolean}     props.isRTL    Check if current locale is RTL.
  *
  * @return {Function} Render function that passes `onScroll` event handler.
  */
-const BlockDraggableWrapper = ( { children } ) => {
+const BlockDraggableWrapper = ( { children, isRTL } ) => {
```

Comment:
As the `RTL` setting is mainly handled by the editor settings, I'm wondering if instead of having the prop `isRTL`, we could directly get it from Redux via the `getSettings` selector within this component. To be honest, both options work for me, so feel free to discard this suggestion if you rather go with the current approach 👍 .

Reply:
Good point, ok, let's stick with this approach 👍 .